dev/dependencies.sh

```diff
@@ -40,7 +40,8 @@ function mvn_build_classpath() {
   $MVN -P$MODULE clean install -DskipTests -am -pl $MVN_MODULES
   $MVN -P$MODULE dependency:build-classpath -am -pl $MVN_MODULES | \
     grep -v "INFO\|WARN" | \
-    # This will skip the first two lines 
+    grep -v "Downloading from" | \
```

Comment:
perhaps we should consider matching the keyword  `[INFO] Dependencies classpath:` and extracting the second line using the `awk` command, as demonstrated in the `sbt_build_client_classpath` function, instead of using `grep -v`. WDYT?

https://github.com/apache/incubator-celeborn/blob/03fc00e6a6177da3a6c8205e4b47551d4c791ca6/dev/dependencies.sh#L60-L63

```shell
./build/mvn dependency:build-classpath -am -pl common
```

<details>

<summary>output</summary>

```
[INFO] --- maven-dependency-plugin:3.1.1:build-classpath (default-cli) @ celeborn-common_2.12 ---
[INFO] Dependencies classpath:
/home/chenfu/.m2/repository/org/slf4j/jcl-over-slf4j/1.7.36/jcl-over-slf4j-1.7.36.jar:/home/chenfu/.m2/repository/io/netty/netty-codec-redis/4.1.93.Final/netty-codec-redis-4.1.93.Final.jar:/home/chenfu/.m2/repository/io/netty/netty-transport-native-epoll/4.1.93.Final/netty-transport-native-epoll-4.1.93.Final-linux-x86_64.jar:/home/chenfu/.m2/repository/org/apache/hadoop/hadoop-client-api/3.2.4/hadoop-client-api-3.2.4.jar:/home/chenfu/.m2/repository/io/netty/netty-codec-dns/4.1.93.Final/netty-codec-dns-4.1.93.Final.jar:/home/chenfu/.m2/repository/io/netty/netty-transport-rxtx/4.1.93.Final/netty-transport-rxtx-4.1.93.Final.jar:/home/chenfu/.m2/repository/org/slf4j/slf4j-api/1.7.36/slf4j-api-1.7.36.jar:/home/chenfu/.m2/repository/io/netty/netty-codec-stomp/4.1.93.Final/netty-codec-stomp-4.1.93.Final.jar:/home/chenfu/.m2/repository/io/netty/netty-resolver-dns-native-macos/4.1.93.Final/netty-resolver-dns-native-macos-4.1.93.Final-osx-aarch_64.jar:/home/chenfu/.m2/repository/org/apache/commons/commons-lang3/3.12.0/commons-lang3-3.12.0.jar:/home/chenfu/.m2/repository/io/netty/netty-transport/4.1.93.Final/netty-transport-4.1.93.Final.jar:/home/chenfu/.m2/repository/io/netty/netty-resolver-dns-native-macos/4.1.93.Final/netty-resolver-dns-native-macos-4.1.93.Final-osx-x86_64.jar:/home/chenfu/.m2/repository/io/dropwizard/metrics/metrics-jvm/3.2.6/metrics-jvm-3.2.6.jar:/home/chenfu/.m2/repository/org/fusesource/leveldbjni/leveldbjni-all/1.8/leveldbjni-all-1.8.jar:/home/chenfu/.m2/repository/io/netty/netty-common/4.1.93.Final/netty-common-4.1.93.Final.jar:/home/chenfu/.m2/repository/io/netty/netty-transport-native-unix-common/4.1.93.Final/netty-transport-native-unix-common-4.1.93.Final.jar:/home/chenfu/.m2/repository/io/dropwizard/metrics/metrics-core/3.2.6/metrics-core-3.2.6.jar:/home/chenfu/.m2/repository/org/apache/htrace/htrace-core4/4.1.0-incubating/htrace-core4-4.1.0-incubating.jar:/home/chenfu/.m2/repository/org/apache/commons/commons-crypto/1.0.0/commons-crypto-1.0.0.jar:/home/chenfu/.m2/repository/io/netty/netty-handler-proxy/4.1.93.Final/netty-handler-proxy-4.1.93.Final.jar:/home/chenfu/.m2/repository/io/netty/netty-codec-xml/4.1.93.Final/netty-codec-xml-4.1.93.Final.jar:/home/chenfu/.m2/repository/io/netty/netty-resolver/4.1.93.Final/netty-resolver-4.1.93.Final.jar:/home/chenfu/.m2/repository/io/netty/netty-transport-native-epoll/4.1.93.Final/netty-transport-native-epoll-4.1.93.Final-linux-aarch_64.jar:/home/chenfu/.m2/repository/io/netty/netty-codec-socks/4.1.93.Final/netty-codec-socks-4.1.93.Final.jar:/home/chenfu/.m2/repository/io/netty/netty-transport-classes-kqueue/4.1.93.Final/netty-transport-classes-kqueue-4.1.93.Final.jar:/home/chenfu/.m2/repository/io/netty/netty-transport-native-kqueue/4.1.93.Final/netty-transport-native-kqueue-4.1.93.Final-osx-x86_64.jar:/home/chenfu/.m2/repository/org/roaringbitmap/RoaringBitmap/0.9.32/RoaringBitmap-0.9.32.jar:/home/chenfu/.m2/repository/com/google/code/findbugs/jsr305/1.3.9/jsr305-1.3.9.jar:/home/chenfu/.m2/repository/io/netty/netty-codec/4.1.93.Final/netty-codec-4.1.93.Final.jar:/home/chenfu/.m2/repository/io/netty/netty-transport-classes-epoll/4.1.93.Final/netty-transport-classes-epoll-4.1.93.Final.jar:/home/chenfu/.m2/repository/io/netty/netty-codec-mqtt/4.1.93.Final/netty-codec-mqtt-4.1.93.Final.jar:/home/chenfu/.m2/repository/io/netty/netty-handler/4.1.93.Final/netty-handler-4.1.93.Final.jar:/home/chenfu/.m2/repository/org/apache/ratis/ratis-proto/2.5.1/ratis-proto-2.5.1.jar:/home/chenfu/.m2/repository/io/netty/netty-codec-memcache/4.1.93.Final/netty-codec-memcache-4.1.93.Final.jar:/home/chenfu/.m2/repository/io/netty/netty-resolver-dns/4.1.93.Final/netty-resolver-dns-4.1.93.Final.jar:/home/chenfu/.m2/repository/io/netty/netty-transport-sctp/4.1.93.Final/netty-transport-sctp-4.1.93.Final.jar:/home/chenfu/.m2/repository/io/dropwizard/metrics/metrics-graphite/3.2.6/metrics-graphite-3.2.6.jar:/home/chenfu/.m2/repository/io/netty/netty-codec-http/4.1.93.Final/netty-codec-http-4.1.93.Final.jar:/home/chenfu/.m2/repository/org/apache/ratis/ratis-common/2.5.1/ratis-common-2.5.1.jar:/home/chenfu/.m2/repository/org/slf4j/jul-to-slf4j/1.7.36/jul-to-slf4j-1.7.36.jar:/home/chenfu/.m2/repository/io/netty/netty-transport-udt/4.1.93.Final/netty-transport-udt-4.1.93.Final.jar:/home/chenfu/.m2/repository/com/google/guava/guava/14.0.1/guava-14.0.1.jar:/home/chenfu/.m2/repository/org/apache/ratis/ratis-thirdparty-misc/1.0.4/ratis-thirdparty-misc-1.0.4.jar:/home/chenfu/.m2/repository/org/scala-lang/scala-library/2.12.15/scala-library-2.12.15.jar:/home/chenfu/.m2/repository/io/netty/netty-buffer/4.1.93.Final/netty-buffer-4.1.93.Final.jar:/home/chenfu/.m2/repository/com/google/protobuf/protobuf-java/3.19.2/protobuf-java-3.19.2.jar:/home/chenfu/.m2/repository/io/netty/netty-resolver-dns-classes-macos/4.1.93.Final/netty-resolver-dns-classes-macos-4.1.93.Final.jar:/home/chenfu/.m2/repository/org/apache/hadoop/hadoop-client-runtime/3.2.4/hadoop-client-runtime-3.2.4.jar:/home/chenfu/.m2/repository/org/apache/ratis/ratis-client/2.5.1/ratis-client-2.5.1.jar:/home/chenfu/.m2/repository/commons-io/commons-io/2.13.0/commons-io-2.13.0.jar:/home/chenfu/.m2/repository/commons-logging/commons-logging/1.1.3/commons-logging-1.1.3.jar:/home/chenfu/.m2/repository/io/netty/netty-transport-native-kqueue/4.1.93.Final/netty-transport-native-kqueue-4.1.93.Final-osx-aarch_64.jar:/home/chenfu/.m2/repository/org/yaml/snakeyaml/1.33/snakeyaml-1.33.jar:/home/chenfu/.m2/repository/io/netty/netty-all/4.1.93.Final/netty-all-4.1.93.Final.jar:/home/chenfu/.m2/repository/org/scala-lang/scala-reflect/2.12.15/scala-reflect-2.12.15.jar:/home/chenfu/.m2/repository/io/netty/netty-codec-smtp/4.1.93.Final/netty-codec-smtp-4.1.93.Final.jar:/home/chenfu/.m2/repository/org/roaringbitmap/shims/0.9.32/shims-0.9.32.jar:/home/chenfu/.m2/repository/io/netty/netty-codec-http2/4.1.93.Final/netty-codec-http2-4.1.93.Final.jar:/home/chenfu/.m2/repository/io/netty/netty-codec-haproxy/4.1.93.Final/netty-codec-haproxy-4.1.93.Final.jar
```
</details>

Reply:
Checked the `maven-dependency-plugin` plugin code, the output will change to a new line, so use `grep -A1`

https://github.com/apache/maven-dependency-plugin/blob/34348ecee0a3588037287df2cad9269e10975c40/src/main/java/org/apache/maven/plugins/dependency/fromDependencies/BuildClasspathMojo.java#L233

```java
getLog().info("Dependencies classpath:" + System.lineSeparator() + cpString);
```

Reply:
> so use grep -A1

seems to be a simpler solution than `sbt_build_client_classpath`

Reply:
`awk` looks more professional and cool.